source/fe/fe_values.cc

```diff
@@ -597,7 +598,7 @@ namespace FEValuesViews
                   const double *shape_value_ptr =
                     &shape_values(shape_function_data[shape_function].row_index[d],0);
                   for (unsigned int q_point=0; q_point<n_quadrature_points; ++q_point)
-                    values[q_point][d] += value **shape_value_ptr++;
+                    values[q_point][d] += value * (*shape_value_ptr++);
```

Comment:
nice change -- I think the meaning of the previous statement was completely obscure!